Makefile

```diff
@@ -109,6 +123,13 @@ $(SETUP_ENVTEST):
 	# see https://github.com/kubernetes-sigs/controller-runtime/tree/master/tools/setup-envtest
 	GOBIN=$(shell pwd)/bin go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
+APPLYCONFIGURATION_GEN := $(shell pwd)/bin/applyconfiguration-gen
+.PHONY: applyconfiguration-gen
+applyconfiguration-gen: $(APPLYCONFIGURATION_GEN) ## Download applyconfiguration-gen locally if necessary
+$(APPLYCONFIGURATION_GEN):
+	# see https://github.com/kubernetes/code-generator/tree/master/cmd/applyconfiguration-gen
+	GOBIN=$(shell pwd)/bin go install k8s.io/code-generator/cmd/applyconfiguration-gen@v0.28.3
```

Comment:
We are using `aqua` to manage packages for `accurate` to automate package updates using `renovate`.
We would like to manage `applyconfiguration-gen` with `aqua` as well.
But that is a bit complicated. I will create a PR for that later.

Reply:
#109